Doc/c-api/intro.rst

```diff
@@ -69,10 +69,12 @@ standard headers) have one of the prefixes ``Py`` or ``_Py``.  Names beginning
 with ``_Py`` are for internal use by the Python implementation and should not be
 used by extension writers. Structure member names do not have a reserved prefix.
 
-**Important:** user code should never define names that begin with ``Py`` or
-``_Py``.  This confuses the reader, and jeopardizes the portability of the user
-code to future Python versions, which may define additional names beginning with
-one of these prefixes.
+.. note::
+
+   User code should never define names that begin with ``Py`` or
```

Comment:
Since you removed the beginning of this line, the whole paragraph probably needs to be reflowed.

Reply:
@benjaminp Done. I made sure the maximum number of words fit per line within the paragraph changed, within the 80 char width limit for reST.